airflow/macros/__init__.py

```diff
@@ -26,8 +26,6 @@
 import dateutil  # noqa
 from pendulum import DateTime
 
-from airflow.macros import hive  # noqa
```

Comment:
This feels like a breaking change. What if we make it accessible via `__getattr__` and just blindly try to import it from the provider? That's probably the right balance here.

Reply:
I had it initially in, but I think it's kinda over-the-top. The macros only really make sense to be use in JINJA2 context (and they will be compatible). I cannot think anyone would use them.

But... it does not hurt eventually so maybe indeed worth adding it.

Reply:
Done.